data_steward/validation/main.py

```diff
@@ -795,6 +870,11 @@ def _get_submission_folder(bucket, bucket_items, force_process=False):
         has been processed and force_process is False or no submission
         directory exists
     """
+
+    # fast exit
+    if bucket_items is None or len(bucket_items) is 0:
```

Comment:
```suggestion
    if not bucket_items:
```

Wanted to put this here while I was thinking about it.  I think it would be "more pythonic" to say "if not bucket_items" than to do the existing double conditional here.